x-pack/plugins/ml/public/application/formatters/format_value.ts

```diff
@@ -53,7 +53,7 @@ export function formatValue(
 // For time_of_day or time_of_week functions the anomaly record
 // containing the timestamp of the anomaly should be supplied in
 // order to correctly format the day or week offset to the time of the anomaly.
-function formatSingleValue(
+export function formatSingleValue(
```

Comment:
Maybe you should change the `mlFunction` to be optional here? Originally this was written for formatting values from anomaly records, but this is a good example of where the function has uses outside of anomaly records. Then you wouldn't need to pass the empty string when calling it from the decision path chart.

Reply:
Updated here https://github.com/elastic/kibana/pull/80448/commits/4c3996242651e19ece5c0ac291743f3d012b24de